insights/parsers/rsyslog_conf.py

```diff
@@ -10,70 +10,76 @@
 
 Due to high parsing complexity, this parser presents a simple line-based
 view of the file that meets the needs of the current rules.
-
-Example:
-    >>> content = '''
-    ... :fromhost-ip, regex, "10.0.0.[0-9]" /tmp/my_syslog.log
-    ... $ModLoad imtcp
-    ... $InputTCPServerRun 10514"
-    ... '''.strip()
-    >>> from insights.tests import context_wrap
-    >>> rsl = RsyslogConf(context_wrap(content))
-    >>> len(rsl)
-    3
-    >>> len(list(rsl))
-    3
-    >>> any('imtcp' in n for n in rsl)
-    True
 """
-from .. import Parser, parser, get_active_lines
-
-import re
+from .. import Parser, parser, get_active_lines, LegacyItemAccess
 from insights.specs import Specs
 
 
 @parser(Specs.rsyslog_conf)
-class RsyslogConf(Parser):
+class RsyslogConf(Parser, LegacyItemAccess):
     """
-    Parses `/etc/rsyslog.conf` info simple lines.
+    Parses `/etc/rsyslog.conf` content.
 
     Skips lines that begin with hash ("#") or are only whitespace.
 
     Attributes:
         data (list): List of lines in the file that don't start
             with '#' and aren't whitespace.
-        config_items(dict): Configuration items opportunistically found in the
-            configuration file, with their values as given.
+
+    Example:
+        >>> type(rsysconf)
+        <class 'insights.parsers.rsyslog_conf.RsyslogConf'>
+        >>> len(rsysconf)
+        13
+        >>> rsysconf[2]
+        'authpriv.*                                              /var/log/secure'
     """
+    def __init__(self, *args, **kwargs):
+        super(RsyslogConf, self).__init__(*args, **kwargs)
 
     def parse_content(self, content):
-        self.data = get_active_lines(content)
+        self.data = []
+        brace_flag = False
+        parenthesis_flag = False
+        parenthesis_string = ""
+        brace_string = ""
 
-        self.config_items = {}
-        # Config items are e.g. "$Word value #optional comment"
-        config_re = re.compile(r'^\s*\$(?P<name>\S+)\s+(?P<value>.*?)(?:\s+#.*)?$')
-        for line in self.data:
+        for line in get_active_lines(content):
             lstrip = line.strip()
```

Comment:
Since `lstrip` is a reserved keyword of python. Would you mind changing it to sth like `l_strip`?